numba/cuda/dispatcher.py

```diff
@@ -809,6 +816,12 @@ def inspect_types(self, file=None):
 
     @property
     def ptx(self):
+        warnings.warn(
```

Comment:
Suggest adding tests to `numba.cuda.tests.cudapy.test_dispatcher` to ensure that these warnings appear correctly (both from the dispatcher object and the kernel object). A pattern to follow might be similar to:
https://github.com/numba/numba/blob/290bcaa63804fb99b1fe238b0f8fe1caf7686899/numba/tests/test_deprecations.py#L16-L29

Reply:
Test is added.